pkgs/applications/virtualization/singularity/generic.nix

```
@@ -219,7 +220,7 @@ in
     '') sourceFilesWithDefaultPaths}
 
     substituteInPlace internal/pkg/util/gpu/nvidia.go \
-      --replace \
+      --replace-warn \
```

```suggestion
      --replace-fail \
```

Warn is never a good idea. If something changes in that specific file, you want it to fail, and then you can fix it yourself.

---

@SomeoneSerge Is this line relevant for Sylabs's Singularity?